tests/Console/ReplayCommandTest.php

```diff
@@ -68,6 +68,16 @@ public function if_no_projectors_are_given_it_will_ask_if_it_should_run_events_a
             ->assertExitCode(0);
     }
 
+    //**/*@test*/*//
```

Comment:
Why the annotation is like this?

Reply:
Seems like my IDE did something funny to the formatting of the annotation, fixed now. Thank you!